src/core/viz/expressions/base.js

```diff
@@ -32,14 +30,18 @@ export default class Base {
         this._getChildren().map(child => child.parent = this);
         this.preface = '';
         this._shaderBindings = new Map();
-        this._uid = uid++;
     }
 
     _bind(metadata) {
         this._compile(metadata);
         return this;
     }
 
+    _setUID(idGenerator){
```

Comment:
this method is called outside so it should be prefixed with an underscore

Reply:
You mean, it should not? Anyways, this is consistent with the other methods in `Base`, and in many other places. We may want to change that, but that's outside the scope.